docs/fast_data/monitoring/dashboards/unmapped_fields.md

```diff
@@ -1,207 +0,0 @@
----
```

Comment:
We did not, I skipped it because I didn't see the dashboard on our grafana. My bad. Just added the new version, more concise and with a grafana json to download